packages/embark-pipeline/src/webpack.config.js

```diff
@@ -44,7 +44,7 @@ const buildDir = path.join(dappPath, embarkJson.buildDir);
 // unexpectedly
 let versions;
 try {
-  versions = glob.sync(path.join(dappPath, '.embark/versions/*/*'));
+  versions = glob.sync(path.join(dappPath, '.embark/versions/*/*/*'));
```

Comment:
Could we go with `.embark/versions/**/*` instead?

Reply:
I could do an experiment with glob, but I'm concerned that `**` might mean "any depth" and I actually do want to go down 3 levels exactly to get all the `node_modules` folders that match `[packageName]/[packageVersion]/node_modules`, so since that's what this does (never goes any deeper)